app/page.tsx

```diff
@@ -118,61 +118,71 @@ export default function Home() {
-              {quizzes.map((data, index) => (
-                <div key={index} className="flex mt-4 items-center" >
-                  <div>
-                    <p className="text-gray-700 text-sm md:text-base mx-6 md:mx-8">
-                      {formatSessionTime(data.sessionOccurrence.start_time)}
-                    </p>
-                    <p className="text-gray-700 text-sm md:text-base mx-6 md:mx-8">
-                      {formatSessionTime(data.sessionOccurrence.end_time)}
-                    </p>
-                  </div>
-                  <div className="bg-card rounded-lg shadow-lg min-h-24 h-auto py-6 relative w-full flex flex-row justify-between mr-4">
-                    <div className={`${index % 2 === 0 ? 'bg-orange-200' : 'bg-red-200'} h-full w-2 absolute left-0 top-0  rounded-s-md`}></div>
-                    <div className="text-sm md:text-base font-semibold mx-6 md:mx-8">
-                      <span className="font-normal pr-4">Subject:</span> {data.sessionDetail.meta_data.stream}
-                      <div className="text-sm md:text-base font-semibold ">
-                        <span className="font-normal pr-9">Type:</span> {data.sessionDetail.meta_data.test_type}
+            {quizzes.length > 0 ? (
+              <div className="grid grid-cols-1 gap-4 pb-40">
+                {quizzes.map((data, index) => (
+                  <div key={index} className="flex mt-4 items-center" >
+                    <div>
+                      <p className="text-gray-700 text-sm md:text-base mx-6 md:mx-8">
+                        {formatSessionTime(data.sessionOccurrence.start_time)}
+                      </p>
+                      <p className="text-gray-700 text-sm md:text-base mx-6 md:mx-8">
+                        {formatSessionTime(data.sessionOccurrence.end_time)}
+                      </p>
+                    </div>
+                    <div className="bg-card rounded-lg shadow-lg min-h-24 h-auto py-6 relative w-full flex flex-row justify-between mr-4">
+                      <div className={`${index % 2 === 0 ? 'bg-orange-200' : 'bg-red-200'} h-full w-2 absolute left-0 top-0  rounded-s-md`}></div>
+                      <div className="text-sm md:text-base font-semibold mx-6 md:mx-8">
+                        <span className="font-normal pr-4">Subject:</span> {data.sessionDetail.meta_data.stream}
+                        <div className="text-sm md:text-base font-semibold ">
+                          <span className="font-normal pr-9">Type:</span> {data.sessionDetail.meta_data.test_type}
+                        </div>
                       </div>
+                      {renderButton(data)}
                     </div>
-                    {renderButton(data)}
                   </div>
-                </div>
-              ))}
-            </div>
+                ))}
+              </div>) : (
+              <p className="text-gray-700 text-sm md:text-base mx-6 md:mx-8">
```

Comment:
i think eventually we can put all classes as variables like in quiz frontend. so that we can edit at one place and can see changes throughout. 
<p className="infoMessageClass">
and infoMessageClass = "text-gray-700 .." etc